compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -1699,8 +1699,8 @@ object Parsers {
           case pt @ Ident(tpnme.WILDCARD_STAR) =>
             migrationWarningOrError("The syntax `x @ _*' is no longer supported; use `x : _*' instead", startOffset(p))
             atPos(startOffset(p), offset) { Typed(p, pt) }
```

Comment:
The `offset` in the above looks dubious too

Reply:
I checked the positions of those trees and they look fine.